srcpkgs/python-ldap3/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'python-ldap3'
+pkgname=python-ldap3
+version=2.5.2
+revision=1
+noarch=yes
+wrksrc=ldap3-${version}
+build_style=python-module
+pycompile_module="ldap3"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
```

Comment:
python-pyasn1 is required, replace python with it, and in the python3-ldap3 package as well